xbmc/rendering/gl/RenderSystemGL.cpp

```diff
@@ -44,6 +44,8 @@ bool CRenderSystemGL::InitRenderSystem()
     m_RenderVersion = ver;
   }
 
+  CLog::Log(LOGERROR, "CRenderSystemGL::%s - Version: %s, Major: %d, Minor: %d", __FUNCTION__, ver, m_RenderVersionMajor, m_RenderVersionMinor);
```

Comment:
What is the reason to log this as an ERROR rather than NOTICE, now we get:
```
2019-03-13 03:40:56.390 T:140102144884032 ERROR: CRenderSystemGL::InitRenderSystem - Version: 4.5 (Core Profile) Mesa 18.3.4, Major: 4, Minor: 5
```
which doesn't look like an error situation too me...

Reply:
Looks like a fuckup from my side.
Will change it to notice.